pkg/operation/care/webhook_remediation_test.go

```diff
@@ -354,6 +359,23 @@ var _ = Describe("WebhookRemediation", func() {
 						},
 						{
 							Name:           "some-webhook3.example.com",
+							TimeoutSeconds: pointer.Int32(10),
```

Comment:
Can you separate the tests for leases to a new `It` or maybe change the names of the webhooks like `some-webhook-with-leases.example.com` or something.